### PR TITLE
Add config for stack rollback on deployment

### PR DESCRIPTION
**Pull request by markdboyd**
PR to run changes from #80 through CI testing

cc @garysieling

### CODE REVIEW

**Comment by garysieling** [comment]
Thanks, do you think there is any chance my changes caused those test failures? If I'm reading the CircleCI output right, it failed on the commit before my changes were merged in:

https://circleci.com/gh/developmentseed/kes/249

**Comment by markdboyd** [comment]
@garysieling Yeah it looks like those failures came immediately from the tip of this branch before your work was merged in. I'm taking a look

**Comment by markdboyd** [comment]
@garysieling The problem wasn't actually related to the code. It was because the test fixtures for deploying Lambdas were using Node v6.10, which AWS has stopped supporting, so the deployments were failing with this error:

```
The runtime parameter of nodejs6.10 is no longer supported for creating or updating AWS Lambda functions. We recommend you use the new runtime (nodejs8.10) while creating or updating functions. (Service: AWSLambdaInternal; Status Code: 400; Error Code: InvalidParameterValueException; Request ID: dbb74cb5-9905-11e9-b66c-574f1151bc74)
```

I updated the fixtures to use Node v8.10 and the tests passed for me, so they should be good now.

File: docs/API.md [hunk 1]
```diff
@@ -418,6 +419,20 @@ replace values of file 1 if they have the same key.
 
 Returns **[String](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)** Merged Yaml file in string format
 
+## loadKesOverride
+
+Attempt to load a Kes override class.
+
+Throw the error if it is something other than that the Kes override
```
Comment (abarciauskas-bgse) [inline docs/API.md]: This phrasing doesn't really make sense to me... should it be:
> Throw an error if the kes override class does not exist
?

Comment (garysieling) [reply docs/API.md]: I'm not 100% sure how this change ended up in this PR, guessing this got merged in from something older...

**Review by abarciauskas-bgse — APPROVED** [review]
One comment on documentation otherwise looks reasonable

**Pull request merged**